specification/columns/commitmentutilizationunit.md

```diff
@@ -0,0 +1,49 @@
+# Commitment Utilization Unit
+
+Commitment Utilization Unit is a String value representing the amortization unit of a [*commitment-based discount*](#glossary:commitment-based-discount) during a billing period.
```

Comment:
I'm more inclined to refer to this as a "commitment unit" rather than an "amortization unit" and explain that a "commitment unit" is the amount of resource consumption or service spend that you committed purchasing for a given commitment discount.

Reply:
Will add a reference to "Commitment Unit" to the glossary as well